cwl/src/main/scala/wdl4s/cwl/package.scala

```diff
@@ -91,4 +91,19 @@ package object cwl extends TypeAliases {
     import wdl4s.cwl.Implicits.enumerationEncoder
     commandLineTool.asJson
   }
+
+  def encodeCwlWorkflow: Workflow => Json = { workflow =>
```

Comment:
I should explain that I like it because 

 * when you are composing functions you don't have to partially apply them. 
 * you don't have to name the argument, i.e. you get a nice underscore to use.
 * It's easier to read IMO